app/assets/stylesheets/application.scss

```diff
@@ -29,26 +30,53 @@ body {
 .bot-avatar {
   border: 6px solid #C63D0F;
   border-radius: 1000px;
-  height: 50%;
-  width: 50%;
+  height: auto;
+  width: 80%;
 }
 
 .user-picture {
   border: 5px solid #3B3738;
   border-radius: 800px;
-  height: 200px;
-  width: 200px;
+  height: auto;
+  width: 65%;
 }
 
 .profile-box {
   background-color: #fdf3e7;
   color: #3B3738;
+  border-style: none;
+
 }
 
 li {
   list-style-type: none;
 }
 
+.bot_box {
+  margin-top: 15%;
+  padding: 2%;
+}
+
+.interaction-box {
+  background-color: #fdf3e7;
```

Comment:
Color literals like `#fdf3e7` should only be used in variable declarations; they should be referred to via variable everywhere else.